vue-docker/vuework03/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM node:lts-alpine3.12
+
+WORKDIR /app
+
+# COPY ./package.json ./yarn.lock /
+
+# COPY . .
+
+COPY . /app
+
+RUN yarn install
```

yarn installに関して
キャッシュが使用できることから、以下のように記述することが多いです。
```suggestion
COPY package.json yarn.lock ./

RUN yarn install

COPY . /app
```
こうすることにより、package.jsonの変更がなかったらキャッシュを使用、変更があればコマンドを実行してくれます。
英語でNode.jsの記事ですが、以下が参考になります。
http://bitjudo.com/blog/2014/03/13/building-efficient-dockerfiles-node-dot-js/